### PR TITLE
iopadmap: Attempt to give new wires/cells meaningful names

### PR DESCRIPTION
**Pull request by rqou**

### CODE REVIEW

File: passes/techmap/iopadmap.cc [hunk 1]
```diff
@@ -366,7 +370,9 @@ struct IopadmapPass : public Pass {
 
 						SigBit wire_bit(wire, i);
 
-						RTLIL::Cell *cell = module->addCell(NEW_ID, RTLIL::escape_id(celltype));
+						RTLIL::Cell *cell = module->addCell(
+							module->uniquify(stringf("$iopadmap$%s.%s", RTLIL::id2cstr(module->name), RTLIL::id2cstr(wire->name))),
```
Comment (mwkmwkmwk) [inline passes/techmap/iopadmap.cc]: log_id?

File: passes/techmap/iopadmap.cc [hunk 2]
```diff
@@ -380,12 +386,16 @@ struct IopadmapPass : public Pass {
 				}
 				else
 				{
-					RTLIL::Cell *cell = module->addCell(NEW_ID, RTLIL::escape_id(celltype));
+					RTLIL::Cell *cell = module->addCell(
+						module->uniquify(stringf("$iopadmap$%s.%s", RTLIL::id2cstr(module->name), RTLIL::id2cstr(wire->name))),
```
Comment (mwkmwkmwk) [inline passes/techmap/iopadmap.cc]: ditto

**Comment by mwkmwkmwk** [comment]
@rqou ping? Do you still intend to merge this PR?

**Comment by rqou** [comment]
updated. sorry, coronavirus has been a huge distraction recently

**Pull request merged**